dashboard/src/pages/hardwareDetails/HardwareDetails.tsx

```diff
@@ -24,7 +34,29 @@ function HardwareDetails(): JSX.Element {
       </Skeleton>
     );
 
-  return <HardwareDetailsTabs HardwareDetailsData={data} />;
+  return (
+    <div className="flex flex-col pt-8">
+      <Breadcrumb>
+        <BreadcrumbList>
+          <BreadcrumbItem>
+            <BreadcrumbLink to="/hardware" search={searchParams}>
```

Comment:
`search={{ previousParams => {...previousParams, searchParams }` 
let's not worry about useMemo for today